language/move-vm/runtime/src/loader.rs

```diff
@@ -812,8 +812,11 @@ impl Loader {
             bytes: Vec<u8>,
             data_store: &mut impl DataStore,
         ) -> VMResult<CompiledModule> {
-            let module = CompiledModule::deserialize(&bytes).map_err(|_| {
+            let module = CompiledModule::deserialize(&bytes).map_err(|err| {
+                error!("[VM] deserializer for module returned error: {:?}", err);
```

Comment:
I think it would also be really useful to log `id` here... that way we'll also know which module this is failing on.

Reply:
Ah, didn't see it passed in above, will do